services/app-web/src/s3/s3Amplify.ts

```diff
@@ -42,7 +42,7 @@ function newAmplifyS3Client(bucketName: string): S3ClientT {
             try {
                 const stored = await Storage.put(`${uuid}.${ext}`, file, {
                     contentType: file.type,
-                    contentDisposition: `attachment; filename=${file.name}`,
```

Comment:
huh i'm surprised the backticks  worked. I would think `attachment; filename="${file.name}"`

Reply:
that might well have worked, too